Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -265,33 +291,73 @@ public function flushByTag(string $tag): int
         }
 
         $script = "
```

Comment:
i would put those in `'LUA'` nowdocs, should have then ide language detection, but never mind ^^